### ISSUE TITLE
Cider refuses to open window

### ISSUE THREAD

**Issue by cmdrneon** [ciderapp/Cider#1003]
### Describe the Bug

Just installed Cider yesterday as an avid Apple Music user and was having issues opening it.

```
[08:38:10.373] [info]  From here can you check for real changes, the initial scan has been completed.
[08:38:10.374] [info]  Cider client port: 9000
[08:38:10.376] [info]  Cider remote port: 6942
[08:38:10.379] [info]  remote broadcasted
[08:38:10.465] [error] Error: Failed to lookup view "main" in views directory "/opt/Cider/resources/app.asar/src/renderer/views"
    at Function.render (/opt/Cider/resources/app.asar/node_modules/express/lib/application.js:597:17)
    at ServerResponse.render (/opt/Cider/resources/app.asar/node_modules/express/lib/response.js:1039:7)
    at /opt/Cider/resources/app.asar/src/main/base/browserwindow.ts:423:17
    at Layer.handle [as handle_request] (/opt/Cider/resources/app.asar/node_modules/express/lib/router/layer.js:95:5)
    at next (/opt/Cider/resources/app.asar/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/opt/Cider/resources/app.asar/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/opt/Cider/resources/app.asar/node_modules/express/lib/router/layer.js:95:5)
    at /opt/Cider/resources/app.asar/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/opt/Cider/resources/app.asar/node_modules/express/lib/router/index.js:346:12)
    at next (/opt/Cider/resources/app.asar/node_modules/express/lib/router/index.js:280:10)
```

### Steps to Reproduce

1. Tried to open the app on a brand new install.

### Environment Information

- OS: Debian 11 (bullseye)
- App Version and/or Commit ID: 1.4.5-beta-2212

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by PolygonHJ** [comment]
I've just experienced the same issue on Manjaro 21 for both:
- AUR 1.4.5-1
- Flatpak 1.4.5.2022

**Comment by MiguelGuthridge** [comment]
I've got the same error

Kubuntu 22.04
Cider v1.4.5.2215 from Flatpak

**Comment by Pixtriks** [comment]
Same issue here after updating Cider. 

Pop_OS 22.04
Flatpak v1.4.5.2215

**Comment by pietropecchi** [comment]
Same issue here after the last update

Fedora 35
Flatpak v1.4.5.2215

i try to use alien to install the .deb version, but get the same error.

The SNAP version looks working fine

**Comment by cryptofyre** [comment]
We’re investigating the issue. Bare with us while we get it resolved.

**Comment by vapormusic** [comment]
Latest Cider build should work now, however we're waiting for the Flatpak maintainer to update the packages

**Comment by pietropecchi** [comment]
on Fedora I again try to use alien to install the .deb version, but get the same error.

I will wait for the Flatpak update

**Comment by cmdrneon** [comment]
Bug has been fixed with 1.4.6-beta-2231
apt/dpkg does say `W: Repository is broken: cider:amd64 (= 1.4.6-beta-2231) has no Size information` after a full reinstall of the package, however the package installs correctly regardless
